boot/boot.js

```diff
@@ -1139,13 +1139,13 @@ $tw.Wiki.prototype.deserializeTiddlers = function(type,text,srcFields) {
 		fields = Object.create(null);
 	if(!deserializer && $tw.config.fileExtensionInfo[type]) {
 		// If we didn't find the serializer, try converting it from an extension to a content type
-		type = $tw.config.fileExtensionInfo[type].type;
```

Comment:
Hello, it is because at the first commit I changed that sentence but in the next commit I restored it.